src/strapp/http/client.py

```diff
@@ -89,6 +89,8 @@ def make_request(
                 requests.exceptions.HTTPError,
                 requests.exceptions.ConnectionError,
                 requests.exceptions.Timeout,
+                requests.exceptions.ConnectionError,
```

Comment:
Seems like we have `ConnectionError` already in line 90 😄

Reply:
Thanks, looks like I had some tunnel vision. 

I think I wasn't looking for it since we do have pure ConnectionErrors in our Sentry tickets 🤔